frontend/src/index.css

```
@@ -1342,3 +1342,590 @@ a:hover { color: var(--ai-600); }
   color: var(--text-tertiary);
   margin: 0;
 }
+
+/* ═══════════════════════════════════════
+   About Page — Client-Facing System Overview
+   ═══════════════════════════════════════ */
+
+@keyframes aboutFadeUp {
```

Stylelint (17.4.0)

[error] Expected keyframe name "aboutFadeUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)